server/api.js

```diff
@@ -1,9 +1,60 @@
 import { Router } from "express";
-
+const passport = require("passport");
+const GithubStrategy = require("passport-github2").Strategy;
 const router = Router();
 
+const GITHUB_CLIENT_ID = "979bf3578c79fbc73e1e";
+
+const CLIENT_URL = "http://localhost:3000/auth/github/callback";
+
+const GITHUB_CLIENT_SECRET = process.env.GITHUB_CLIENT_SECRET;
+
+
+
+passport.serializeUser((user, done) => {
+	done(null, user);
+});
+
+passport.deserializeUser((user, done) => {
+	done(null, user);
+});
+
+passport.use(
+	new GithubStrategy(
+		{
+			clientID: GITHUB_CLIENT_ID,
+			clientSecret: GITHUB_CLIENT_SECRET,
+			callbackURL: CLIENT_URL,
+		},
+		function (accessToken, refreshToken, profile, done) {
+			done(null, profile);
+		}
+	)
+);
+
 router.get("/", (_, res) => {
 	res.json({ message: "world" });
 });
 
+router.get("/login", function (req, res) {
```

Comment:
I don't think we need this?

Reply:
thank you Sam , i deleted this block line 39